### DISCUSSION TITLE
Color Management weird when sampling image textures

### DISCUSSION THREAD

**Discussion by Kolupsy** [bnpr/Malt#213] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88381426/147364393-81a863e9-e816-49c3-a4f9-3fc215a2bb3b.png)
I tried sampling a texture using the sampler2D_sample shader function. The colors turned out a lot different than the image itself is. The image is a 16bit RGB TIFF (as seen on the image). How can I get my setup to display the proper colors?

**Comment by pragma37** [comment]
Color Management is handled by Blender. Do you maybe have Filmic enabled?
It may also be caused by the Viewport Bit Depth being 8 bits? You can change it in the World settings to 32.

If that's not the case, then upload a .blend file to reproduce the issue.

**Comment by Kolupsy** [comment]
[color_management_issue.zip](https://github.com/bnpr/Malt/files/7775063/color_management_issue.zip)
I used the Standard color management when baking the texture as well as when coming across the isse rendering in Malt.
Changing the Bit Depth did not change anything about that (from what I can see). The above zip contains an example of that problem.

**Comment by pragma37** [comment, marked as answer]
There was indeed a bug.
It's already fixed, see #215 for reference.
